testsuite/integration/basic/pom.xml

```diff
@@ -884,7 +884,6 @@
                                 <include>org/wildfly/test/integration/microprofile/**/*Test.java</include>
                             </includes>
                             <excludes>
-                                <!-- requires org.apache.commons.lang in test servlet otherwise passes -->
```

Comment:
This should be restored and just changed to reference lang3.

If we keep the exclude element we need the comment explaining why.